tests/typecheck/fields/test_related.yml

```
@@ -640,6 +640,51 @@
                   pass
 
 
+-   case: test_related_managers_for_subclass_with_multiple_inheritance
+    main: |
+        from myapp import models
+    installed_apps:
+        - myapp
+    files:
+        -   path: myapp/__init__.py
+        -   path: myapp/models.py
+            content: |
+                from django.db import models
+                class TimestampedModel(models.Model):
+                    id = models.UUIDField(primary_key=True, editable=False)
+                    created_at = models.DateTimeField(auto_now_add=True, db_index=True)
+                    updated_at = models.DateTimeField(auto_now=True, db_index=True)
+                    class Meta:
+                        abstract = True
+
+                class User(TimestampedModel):
+                    name = models.TextField()
+
+                def DynamicManager():
+                    class InnerManager(models.Manager):
+                        def some_method(self, *args, **kwargs):
+                            pass
+
+                    return InnerManager()
+
+                class Booking(TimestampedModel):
+                    renter = models.ForeignKey(User, on_delete=models.PROTECT)
+                    owner = models.ForeignKey(User, on_delete=models.PROTECT, related_name='bookingowner_set')
+
+                    class Meta(TimestampedModel.Meta):
+                        ordering = ['-created_at']
+
+                    objects = DynamicManager()
+
+                def process_booking(user: User):
+                    reveal_type(user.bookingowner_set)
+                    reveal_type(user.booking_set)
+    out: |
+        myapp/models:9: error: Couldn't resolve related manager for relation booking, constructing a default one instead.
+        myapp/models:9: error: Couldn't resolve related manager for relation bookingowner_set, constructing a default one instead.
```

Thought: I'm wondering if it could be valuable to try to include the path/fullname to the relation declaration as well?

It's good that the name of the reverse relation is there. But maybe it's also nice if e.g. `myapp.models.Booking.x` was in the message too?

e.g.

```suggestion
        myapp/models:9: error: Couldn't resolve related manager for relation booking, constructing a default one instead. (myapp.models.Booking.renter)
        myapp/models:9: error: Couldn't resolve related manager for relation bookingowner_set, constructing a default one instead. (myapp.models.Booking.owner)
```

I think we have that context/data available when emitting, right? Otherwise just skip this.